### DISCUSSION TITLE
Newbie Question

### DISCUSSION THREAD

**Discussion by JustusJonass231** [Stellarium/stellarium#1979] [General]
Hello Together,

I have installed the Linux Version on my Server but i try to acces the Webview but nothing is shown the Remote Acces Tool is working well. I want to build a Star Register with Custom Names on it. My Programming Skills are not verry good hope some one can help me

**Comment by gzotti** [comment]
What is the purpose of your "Star Register with Custom Names"?

**Reply by JustusJonass231** [reply]
i want to build a skyline with names for a marriage

**Comment by gzotti** [comment]
For Stellarium Desktop, you could create a 'Personal' skyculture. The process is described in the User Guide.
For Stellarium-web, I cannot give instructions. Open source is open to study.

**Reply by JustusJonass231** [reply]
My Problem is i want to creat indiviual cards for the marrigae with names and later i want to show as a akt on a Big Screen the Map and the Stars and then all will scan a code and see the stars and on the big screen the Stars what we have created i have uploaded it to my Linux server thats not the Problem but when i open the for example 192.2482.23424 then the remote tool shows direklty but the web view is not displayed.

Can maybe some one Help me with it i can pay a small amount for its not for buisness is only for a personal use
